internal/pkg/routes/route.go

```diff
@@ -22,14 +22,22 @@ type route struct {
 	subtopic  string
 	logger    logger.Logger
 	pub       publish.Publisher
+	workers   int
+	messages  chan *nats.Msg
+	sub       *nats.Subscription
 }
 
+var (
+	workers = 200
+)
```

Comment:
Use `const workers  = 200`. Also, please move const declarations to the top of the file.